app/models/user.rb

```diff
@@ -89,6 +89,10 @@ def is_school_user?
     school.present?
   end
 
+  def is_la_funded_user?
```

Comment:
Again we get some RSpec syntactic sugar if we replace `is_la_funded_user?` with `la_funded_user?`